apps/communications/dialer/test/marionette/add_contact_PO.js

```diff
@@ -0,0 +1,42 @@
+/* global require, marionette, setup, suite, test */
+'use strict';
+
+//var assert = require('chai').assert;
+
+var Dialer = require('./lib/dialer.js');
+var KeypadView = require('./lib/views/keypad/views.js');
+
+marionette('Dialer > Keypad', function() {
+
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var dialerApp;
+  var actions;
+
+  setup(function() {
+    actions = client.loader.getActions();
+    dialerApp = Dialer.create(client);
+    dialerApp.launch();
+  
+   });
+
+  suite('Test Suite', function() {
+
+    test('Add a contact', function() {
+        
+        var keypadView = new KeypadView(client);
+        keypadView.typeNumber();
```

Comment:
That's probably what you had in mind: [Like in python](https://github.com/mozilla-b2g/gaia/blob/b010065e59936ef13381758637eb69db8be40e14/tests/python/gaia-ui-tests/gaiatest/apps/phone/regions/keypad.py#L25) I'd give a string to type the phone number.